client/extensions/woocommerce/app/settings/shipping/index.js

```diff
@@ -24,28 +25,34 @@ import { isWcsEnabled } from 'woocommerce/state/selectors/plugins';
 
 class Shipping extends Component {
 	state = {
-		pristine: true,
+		pristine: { units: true, shipping: true },
```

Comment:
instead of tracking state in an object, should we consider just having two attributes `unitsClean/Dirty` and `shippingClean/Dirty` - or something like that?

Reply:
I had it in two separate attributes. I have change it to ^ because I liked how it is now encapsulated under one element. It adds a level of tidiness IMO :). It is a matter of personal preferences so I don't mind to change it to separate attributes. @timmyc should I change it?

Reply:
Nope fine as-is, just was a thought when reading the code.